### ISSUE TITLE
webui is not showing same meta data as api

### ISSUE THREAD

**Issue by adrianschroeter** [openSUSE/open-build-service#1401]
The webui shows a different meta data content as the api. We may broke the caching?

Real life example is atm in home:Guillaume_G:branches:Application:Geo

**Comment by hennevogel** [comment]
Can you be a little bit more specific? What's the difference?

**Comment by adrianschroeter** [comment]
The <publish> <disable/>  </publish> part is missing in this example

**Comment by adrianschroeter** [comment]
however, Guillaume has change it now and removed that part. So you can not use it as example anymore.

**Comment by hennevogel** [comment]
So how can one reproduce this?

**Comment by adrianschroeter** [comment]
On Tuesday 24 November 2015, 06:35:04 wrote Henne Vogelsang:
> So how can one reproduce this?

no idea, but I would double check the cache handlers and invalidation ...

**Comment by afaerber** [comment]
See also similar debuginfo confusion: [boo#956525](https://bugzilla.opensuse.org/show_bug.cgi?id=956525)

**Comment by repagh** [comment]
I don't know whether it is the exact same problem, but here is an example

![webstate](https://cloud.githubusercontent.com/assets/2794509/11467438/60c3eb38-9748-11e5-8227-2a6a9ccce941.png)

SLE_11_SP3 was disabled in that example, I selected "take default" through the webui (it should take disabled default, but somehow does not show it). This screenshot is shown after the action, and also after a page refresh. 

This is what viewing the meta through osc gives me:
```
[repa@tud277466 ~]$ osc meta pkg Education wings 
<package name="wings" project="Education">
  <title>Wings3D</title>
  <description>Wings 3D is a subdivision modeler with an user interface that is easy
to use for both beginners and advanced users (inspired by Nendo and
Mirai from Izware).</description>
  <build>
    <disable repository="Fedora_19"/>
    <disable repository="SLE_11_SP3"/>
    <disable repository="Fedora_20"/>
    <disable repository="openSUSE_12.2"/>
    <disable repository="openSUSE_12.3"/>
    <disable repository="openSUSE_13.1"/>
    <disable repository="openSUSE_Evergreen_11.4"/>
    <disable repository="openSUSE_Tumbleweed"/>
    <disable/>
    <enable repository="openSUSE_Factory"/>
    <enable repository="SLE_12"/>
    <enable repository="openSUSE_13.2"/>
  </build>
</package>
```
The explicit disable on SLE_11_SP3 is still there. 

- I think that somehow the webui is not actually updating the back-end *and* it is updating & reflecting the state of a cache
- If I take another browser, the webui information matches between the two browsers
- If I change a setting through osc, the new state is reflected after a refresh of the webui
- Also the webui options have regressed. In this example the shown options for a defaulted item is "disable", while it should be "enable"/"explicitly disable"

May this helps ...

**Comment by hennevogel** [comment]
Fixed  since fa1c1f22dc9c2ef7b25e02989d382da33e237f22
